BLSQ/iaso
# IA-2050 / POLIO-998: remove backdrop click component from IASO + allow backdrop click on dialog

- `BackdropClickModal` component can be moved to blsq-component
- New Dialog is not closing the dialog on backdrop click

Related JIRA tickets : IA-2050, POLIO-998
## Self proofreading checklist


## Changes

- moved code to blsq-component
- adapt blsq-component SimpleDialog to allow backdrop click

## How to test

- Open a storage detail and open status dialog, you should be able to close it by clicking on the background
- retest this feature => https://github.com/BLSQ/iaso/pull/566


## Print screen / video



https://user-images.githubusercontent.com/12494624/236143818-6389260d-4fe2-4e01-af2d-2e5df6ea0d8d.mov



## Notes

This is going along with this [PR](https://github.com/BLSQ/bluesquare-components/pull/115)
